include/l3.h

```diff
@@ -55,4 +55,7 @@ void l3_log_simple(const char *msg, const uint64_t arg1, const uint64_t arg2);
  *
  * Like l3_log_simple() but does not take arguments. Only available on x86-64.
  */
+ #ifdef __cplusplus
+extern "C"
+#endif
```

Comment:
Need this when `l3.c` is linked with C++ binaries using g++. 

Otherwise we will get an unresolved reference to this extern fn.